services/node-status/tsconfig.json

```diff
@@ -0,0 +1,29 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "downlevelIteration": true,
+    "lib": ["dom", "dom.iterable", "esnext"],
+    "allowJs": true,
+    "skipLibCheck": true,
```

Comment:
Would there be a prob if we'd remove the skipLibCheck?

Reply:
Not sure. NextJs adds it by default when creating a new typescript project